### ISSUE TITLE
Training configuration for DocT5query

### ISSUE THREAD

**Issue by jihyukkim-nlp** [beir-cellar/beir#122]
Hi all, thank you for sharing this awesome repo!

I am wondering if the training configuration (e.g., batch size, learning rate, iterations, etc) for 
* [BeIR/query-gen-msmarco-t5-large-v1](https://huggingface.co/BeIR/query-gen-msmarco-t5-base-v1)
* [BeIR/query-gen-msmarco-t5-large-v1](https://huggingface.co/BeIR/query-gen-msmarco-t5-large-v1)
can be shared in addition to the checpoints.

Best regards,
Jihyuk
